storage/http.js

```diff
@@ -0,0 +1,147 @@
+var rx = 0
+var connectionLimit = 10
+
+module.exports = function (root) {
+  var controllers = {}
+  var callbacks = {} // store leaked callbacks here so they may be resumed later
+  var active = {}
+  var queue = []
+  var pending = 0
+  var cache = {}
```

Comment:
Previous `data` variable was inside the `storageFn` closure, so had to become a cache with `name` as key.